### PR TITLE
2674/plp issue

### PR DESCRIPTION
**Pull request by tatiana-scandi**
Issue: https://github.com/scandipwa/scandipwa/issues/2674

### CODE REVIEW

File: packages/scandipwa/src/component/ProductActions/ProductActions.container.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ import {
     CONFIGURABLE,
     DOWNLOADABLE,
     GROUPED,
-    SIMPLE
+    SIMPLE, VIRTUAL
```
Comment (zans-laksa) [inline packages/scandipwa/src/component/ProductActions/ProductActions.container.js:23]: Please place `VIRUAL` constant on a new line.

**Review by zans-laksa — CHANGES_REQUESTED** [review]
Code review not passed.

**Comment by aleksandrakorolova** [comment]
Still can reproduce 
Screenshot 2021-06-30 at 08 48 49

**Review by zans-laksa — APPROVED** [review]
Code review passed.

**Pull request merged**